src/style.css

```
@@ -1,3 +1,61 @@
+@import url("https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap");
+
+:root {
+  --active-color: rgba(0, 172, 220, 1);
+  --main-color: rgba(0, 0, 0, 1);
+}
+
 body {
   margin: 0;
+  box-sizing: border-box;
+}
+
+html {
+  min-width: 1024px;
+  font-family: "Roboto", sans-serif;
+}
+
+.nav {
+  position: relative;
+}
+
+.nav__link {
+  font-size: 12px;
+  color: var(--main-color);
+  text-decoration: none;
+  cursor: pointer;
+  display: block;
+  margin-right: 20px;
+  text-transform: uppercase;
+}
+
+.nav__list {
+  list-style: none;
+  display: flex;
+  padding: 0;
+
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  margin: 10px 50px;
+}
+
+.nav__link:hover {
+  color: var(--active-color);
+}
+
+.is-active {
+  color: var(--active-color);
+}
```

you may list selectors if add the same styles:
```suggestion
.nav__link:hover,
.is-active {
  color: var(--active-color);
}
```